src/LogExpert/LogExpert.csproj

```diff
@@ -109,7 +109,15 @@
     <Compile Include="Dialogs\LogGridCell.cs" />
     <Compile Include="Dialogs\LogTextColumn.cs" />
     <Compile Include="Extensions\ComboBoxExtensions.cs" />
+    <Compile Include="Extensions\MenuStripExtension.cs" />
+    <Compile Include="Extensions\LineToolStripSeparatorExtension.cs">

```

Comment:
Could you please move the Winform Extensions to the folder "Extensions/Forms" just to separate them from the other extensions 😃

Reply:
Done